src/component/utility/export.ts

```diff
@@ -335,17 +335,32 @@ async function resolveBlob(b: Blob): Promise<Blob> {
 
 async function writeImageToClipboard(image: Blob, isSafari = false) {
   await navigator.clipboard.write([
-    new ClipboardItem({ [image.type]: isSafari ? resolveBlob(image) : image }),
+    new ClipboardItem({
+      [image.type]: isSafari ? resolveBlob(image) : image,
+    }),
   ]);
 }
 async function copyBlobToClipboard(canvas: OffscreenCanvas) {
+  // Check if the document is not focused. Focus the window.
+  if (!document.hasFocus()) {
+    window.focus();
+  }
```

Comment:
Is there a way to reproduce this? In my tests with Chrome, it doesn't bring the focus back.

Reply:
You can disable this code and try Luc's scenario (https://github.com/cheminfo/nmrium/issues/3394): switch to another application immediately after clicking the "Copy" button on the "Copy to Clipboard" dialog.

Reply:
yes you are right, it should be removed